### ISSUE TITLE
quasar -m electron --devtools starts application with empty localStorage (no items in it)

### ISSUE THREAD

**Issue by johanna-herrmann** [quasarframework/quasar#12696]
### What happened?

TL;DR:
using ```quasar dev -m electron --devtools``` causes application to start with an empty localStorage.
If ```quasar dev -m electron``` (without --devtools) is used on next start, all items created before are restored
(the items of previous ```quasar dev -m electron``` are restored,
not the items from previous ```quasar dev -m electron --devtools``` start).
--> Seems like with --devtools an independent and empty localStorage is used, new and empty at each start,
but without --devtools the localStorage of previous starts (also without --devtools) is used.

On detail:

In my quasar-electron application i make use of the localStorage (https://www.w3schools.com/html/html5_webstorage.asp).

On the beginning i used ```quasar dev -m electron``` to start my application.
I added an item in localStorage (test=test123).
I checked if the item was there. And yes, it was there.
Also after restarting the application the items is still there.

Then later on development i decided to have vue devtools.
So I changed from ```quasar dev -m electron``` to ```quasar dev -m electron --devtools``` like stated here: https://quasar.dev/quasar-cli/developing-electron-apps/build-commands#vuejs-devtools
The vue devtools opened, then my application opened, but with a localStorage without any item.

I added a new item on localStorage (test2=456test) (see the difference)
But after a restart of the application, the localStorage was empty again.

After some tests and research and no solution, I went back to ```quasar dev -m electron``` (without --devtools).
Know my application started with a localStorage, not empty.
Interestingly, the item in localStorage was not test2=456, but test=test123, so my old item created at the beginning was there again.

Know i made some tests.
With ```quasar dev -m electron```,
I started the application repeatedly and on each start i created exactly one item in localStorage, storing an counter from one to ten.
So my items after ten starts were from test1=one to test10=ten.
At each start i could see all my items.
Then i went to start my application with ```quasar dev -m electron --devtools```
Again, on each start i created exactly one item, counting from eleven to twenty this time.
So my created items now were from test11=eleven to test20=twenty.
At each start I could see the localStorage was empty, until I created the new item.
I went back to ```quasar dev -m electron```, started the application and saw:
the items from test1=one to test10=ten.

My conclusion:
With --devtools an independent and empty localStorage is used, new and empty at each start,
but without --devtools the localStorage of previous starts (also without --devtools) is used, like expected.

### What did you expect to happen?

I would expect ```quasar dev -m electron --devtools``` to do exactly the same like ```quasar dev -m electron``` in this context,
using an not-empty localStorage, also used at the previous start.

### Reproduction URL

https://github.com/diamory/diamory-local-sync

### How to reproduce?

Notice: I used a github repo, cause all the sandboxes above have no ability to start the devtools at all.

To reproduce:
1. Clone repo: https://github.com/diamory/diamory-local-sync
2. Run ```yarn``` to install dependencies
3. Goto package.json and change scripts->dev to ```quasar -m electron --devtools```, if deviating.
4. Run ```yarn dev```
5. Create items in localStorage (you can use devtools's console for it)
6. Stop Application and start again (same command)
7. See the localStorage empty
8. Goto package.json and change scripts->dev to ```quasar -m electron```
9. Run ```yarn dev```
10. See localStorage is empty
11. Create some items in localStorage
12. Restart application (close window and run ```yarn dev``` again)
13. See: your items are still there.

13th is the expected behaviour, 7th is not.

### Flavour

Quasar CLI (@quasar/cli | @quasar/app)

### Areas

Electron Mode

### Platforms/Browsers

Electron

### Quasar info output

```shell
Operating System - Linux(5.13.0-30-generic) - linux/x64
NodeJs - 14.18.0

Global packages
  NPM - 8.0.0
  yarn - 1.22.15
  @quasar/cli - 1.2.2
  @quasar/icongenie - 2.4.2
  cordova - 11.0.0

Important local packages
  quasar - 2.5.5 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app - 3.3.3 -- Quasar Framework local CLI
  @quasar/extras - 1.12.5 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.31 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.0.13
  vuex - Not installed
  electron - 17.1.0 -- Build cross platform desktop apps with JavaScript, HTML, and CSS
  electron-packager - Not installed
  electron-builder - 22.14.13 -- A complete solution to package and build a ready for distribution Electron app for MacOS, Windows and Linux with “auto update” support out of the box
  @babel/core - 7.17.5 -- Babel compiler core.
  webpack - 5.70.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.7.4 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.5.5 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - mark-Lenovo-H535
  enp1s0 - 192.168.0.143
```

### Relevant log output

```shell
Dev mode.......... electron
 Pkg quasar........ v2.5.5
 Pkg @quasar/app... v3.3.3
 Pkg webpack....... v5
 Debugging......... enabled

 Configured browser support (>= 87.57% of global marketshare):
 · Chrome for Android >= 98
 · Firefox for Android >= 96
 · Android >= 98
 · Chrome >= 90
 · Edge >= 95
 · Firefox >= 88
 · iOS >= 12.0-12.1
 · Opera >= 79
 · Safari >= 13

 App • Chaining "Renderer" Webpack config
 App • Chaining "Preload" Webpack config
 App • Chaining "Main" Webpack config
 App • Booting up remote Vue Devtools...
 App • Running "/home/mark/diamory/repositories/diamory-local-sync/node_modules/@vue/devtools/bin.js "

 App •  WAIT  • Compiling of "Renderer" in progress...
 App •  DONE  • "Renderer" compiled with success • 10556ms



 App •  READY  • Compiled: "Renderer"

 App •  WAIT  • Compiling of "Main" in progress...
 App •  WAIT  • Compiling of "Preload" in progress...
 App •  DONE  • "Main" compiled with success • 324ms
 App •  DONE  • "Preload" compiled with success • 354ms



 App •  READY  • Compiled: "Renderer", "Main", "Preload"

 App • Running "/home/mark/diamory/repositories/diamory-local-sync/node_modules/electron/dist/electron --inspect=5858 /home/mark/diamory/repositories/diamory-local-sync/.quasar/electron/electron-main.js"

Debugger listening on ws://127.0.0.1:5858/23c85744-a9c6-49ed-b87f-4869e1583cf8
For help, see: https://nodejs.org/en/docs/inspector
[23700:0307/210946.281661:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
```

### Additional context

- I used a github repo, because all the sandboxes above have no ability to start the vue devtools at all.
- I'm not sure if this problem is only relevant in electron mode
- I searched for solutions on google for some days. no success.
- I also tried to run ```quasar dev -m electron --devtools``` directly, instead of using yarn command. no difference.
